File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Converting PowerPoint Presentation to PDF |Syncfusion|
-description: This section illustrates how to convert PowerPoint Presentation document to PDF; PowerPoint Presentation conversion
+title: Converting PPT Presentation to PDF in Fileformat PDF |Syncfusion
```

Comment:
Control name is wrong, check and change in all places.

Reply:
changed